Do not append '$' as this seem not to work, at least when data provider is in use.

diff --git a/autoload/test/php/codeception.vim b/autoload/test/php/codeception.vim
index d627dd5..ef2b900 100644
--- a/autoload/test/php/codeception.vim
+++ b/autoload/test/php/codeception.vim
@@ -13,7 +13,7 @@ function! test#php#codeception#build_position(type, position) abort
   if a:type ==# 'nearest'
     let testname = s:nearest_test(a:position)
     let filename = a:position['file']
-    if !empty(testname) | let filename .= '::' . testname . '$' | endif
+    if !empty(testname) | let filename .= '::' . testname | endif
     return [filename]
   elseif a:type ==# 'file'
     return [a:position['file']]
